components/AudioPlayer.js

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useRef, useEffect } from 'react'
+import styles from "../components/AudioPlayer.module.css";
+import utilsStyles from '../styles/utils.module.css';
+import Image from 'next/image';
+import { BsArrowLeftShort } from "react-icons/bs"
+import { BsArrowRightShort } from "react-icons/bs"
```

Comment:
```suggestion
import { BsArrowLeftShort, BsArrowRightShort } from "react-icons/bs"
```
You can combine those and make them shorter like above.